### PR TITLE
Religion files

### PR DESCRIPTION
**Pull request by IhateTrains**
Credits to lukaslunt for the files.

### CODE REVIEW

File: ImperatorToCK3/Copy_Files_Release.bat [hunk 1]
```diff
@@ -0,0 +1,6 @@
+echo on
```
Comment (Idhrendur) [inline ImperatorToCK3/Copy_Files_Release.bat:1]: You should also create linux version of this.

File: ImperatorToCK3/Data_Files/blankMod/dummy.txt [hunk 2]
```diff
@@ -1 +0,0 @@
-I'm a dummy file. My purpose is to ensure my directory is created.
```
Comment (Idhrendur) [inline ImperatorToCK3/Data_Files/blankMod/dummy.txt:1]: Was this file left? You should be able to fully delete it now that there are other files.

**Review by Idhrendur — APPROVED** [review]

File: ImperatorToCK3/Data_Files/blankMod/dummy.txt (hunk 2 above)
Comment (IhateTrains) [reply ImperatorToCK3/Data_Files/blankMod/dummy.txt:1]: It's removed

File: ImperatorToCK3/Copy_Files_Release.bat (hunk 1 above)
Comment (IhateTrains) [reply ImperatorToCK3/Copy_Files_Release.bat:1]: I think this is already done in Copy_Files.sh
`
rm -rf "Release-Linux/blankMod"

cp -r "Data_Files/blankMod" "Release-Linux/"
`

**Pull request merged**